docs/source/libraries/xgboost.rst

```diff
@@ -5,15 +5,21 @@ XGBoost
 
 XGBoost_ is a popular Gradient Boosting library with Python interface.
 eli5 supports :func:`eli5.explain_weights` and :func:`eli5.explain_prediction`
-for XGBClassifer_ and XGBRegressor_ estimators. It is tested for
+for XGBClassifer_, XGBRegressor_ and Booster_ estimators. It is tested for
 xgboost >= 0.6a2.
 
 .. _XGBoost: https://github.com/dmlc/xgboost
 .. _XGBClassifer: https://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.XGBClassifier
 .. _XGBRegressor: https://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.XGBRegressor
+.. _Booster: http://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.Booster
 
-:func:`eli5.explain_weights` uses feature importances. Additional
-arguments for XGBClassifer_ and XGBRegressor_:
+:func:`eli5.explain_weights` uses feature importances.
+There is one additional required argument for Booster_ estimator:
+
+* ``is_regression`` True if solving a regression problem
```

Comment:
Why is this argument required? It seems for explain_weights it only affects `Explanation.is_regression` flag